### PR TITLE
lodash: adds missing export for conformsTo and defaultTo

### LINKED ISSUE #12062: _.defaultTo missing in lodash typings
- I tried using the latest `lodash/index.d.ts` file in this repo and had problems.

I could not find a type definition for _.defaultTo.  I think it needs to be added.
- The authors of that type definition are @bczengel, @chrootsu, @stepancar.

### PR DESCRIPTION
**Pull request by tomc974**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

`tsc` type-checks the definitions fine but `npm run lint lodash` reports hundreds of errors which has nothing to do with this PR. Unfortunately, I don't have the time to fix them.

Fixes #12062

### CODE REVIEW

**Comment by tomc974** [comment]
@bczengel  @chrootsu @stepancar @ericanderson @aj-r @Ailrun 

Could some of you please review this PR? and sorry for all the merge conflicts, I completely forgot to sync my repo before submitting this... :sweat:

File: types/lodash/tsconfig.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         ],
         "noImplicitAny": true,
         "noImplicitThis": true,
+        "strictFunctionTypes": false,
```
Comment (aj-r) [inline types/lodash/tsconfig.json]: We support `strictFunctionTypes` now - there is no need to set it to `false`. In fact, it is set to `true` 2 lines down. However if you want to move that line up here (so they are in alphabetical order) I won't object.

**Review by aj-r — CHANGES_REQUESTED** [review]

**Comment by tomc974** [comment]
@aj-r Thanks for the suggestion. Another mistake I could have avoided had I sync my repo before making the PR. Hopefully, I will learn my lesson this time! :smile:

I have a little question though, since I sync'ed with master, when I run `npm run lint lodash`, instead of the lint errors, I receive this error:
> TypeError: ts.isVariableStatement is not a function

Any idea on how to fix that?

**Review by aj-r — APPROVED** [review]
I'm not sure what `ts.isVariableStatement is not a function` means. It must only be an issue on your local machine, since Travis doesn't have any issues. Maybe your `dtslint` is out of date? You could try deleting the `node_module/dtslint` folder and running `npm install` again.

**Comment by tomc974** [comment]
@aj-r Thanks, re-installing dependencies did fix the issue!

**Pull request merged**